File: Fisica/src/Fisica/Models/TopicoForumModel.cs
namespace Fisica.Models
{
    public class TopicoForumModel
    {
        public long Id { get; set; }
        public string Titulo { get; set; }
        public string Descricao { get; set; }
        public string UsuarioCadastro { get; set; }
        public DateTime DataCadastro { get; set; }

        public IList<RespostaTopicoModel>? Respostas { get; set; }
    }
}

File: Fisica/src/Fisica.Website/ClientApp/src/app/shared/models/topico-forum.model.ts
import { RespostaTopico } from "./resposta-topico.model";

export class TopicoForum {
  constructor(
    public id?: number,
    public titulo?: string,
    public descricao?: string,
    public usuarioCadastro?: string,
    public dataCadastro?: Date,
    public respostas?: RespostaTopico[]
  ) { }
}
